doc/nrf/known_issues.rst

```diff
@@ -1849,7 +1898,7 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
```

Comment:
NCSDK-11195: Configuring the BUILD_WITH_TFM configuration SES project configuration or west -t menuconfig results in build errors.
Workaround: Set CONFIG_BUILD_WITH_TFM=y in project configuration file (prj.conf) or through west command line (west build – -DCONFIG_BUILD_WITH_TFM=y)
Affects: from 1.6

Reply:
Added with a heading. Please check.